doc/classes/Viewport.xml

```diff
@@ -519,54 +519,56 @@
 		</constant>
-		<constant name="DEBUG_DRAW_SSIL" value="13" enum="DebugDraw">
+		<constant name="DEBUG_DRAW_SSIL" value="14" enum="DebugDraw">
 			Draws the screen-space indirect lighting texture instead of the scene so that you can clearly see how it is affecting objects. In order for this display mode to work, you must have [member Environment.ssil_enabled] set in your [WorldEnvironment].
 		</constant>
-		<constant name="DEBUG_DRAW_PSSM_SPLITS" value="14" enum="DebugDraw">
+		<constant name="DEBUG_DRAW_PSSM_SPLITS" value="15" enum="DebugDraw">
 			Colors each PSSM split for the [DirectionalLight3D]s in the scene a different color so you can see where the splits are. In order, they will be colored red, green, blue, and yellow.
 		</constant>
-		<constant name="DEBUG_DRAW_DECAL_ATLAS" value="15" enum="DebugDraw">
+		<constant name="DEBUG_DRAW_DECAL_ATLAS" value="16" enum="DebugDraw">
 			Draws the decal atlas used by [Decal]s and light projector textures in the upper left quadrant of the [Viewport].
 		</constant>
-		<constant name="DEBUG_DRAW_SDFGI" value="16" enum="DebugDraw">
+		<constant name="DEBUG_DRAW_SDFGI" value="17" enum="DebugDraw">
 		</constant>
-		<constant name="DEBUG_DRAW_SDFGI_PROBES" value="17" enum="DebugDraw">
+		<constant name="DEBUG_DRAW_SDFGI_PROBES" value="18" enum="DebugDraw">
 		</constant>
-		<constant name="DEBUG_DRAW_GI_BUFFER" value="18" enum="DebugDraw">
+		<constant name="DEBUG_DRAW_GI_BUFFER" value="19" enum="DebugDraw">
 		</constant>
-		<constant name="DEBUG_DRAW_DISABLE_LOD" value="19" enum="DebugDraw">
+		<constant name="DEBUG_DRAW_DISABLE_LOD" value="20" enum="DebugDraw">
 		</constant>
-		<constant name="DEBUG_DRAW_CLUSTER_OMNI_LIGHTS" value="20" enum="DebugDraw">
+		<constant name="DEBUG_DRAW_CLUSTER_OMNI_LIGHTS" value="21" enum="DebugDraw">
 		</constant>
-		<constant name="DEBUG_DRAW_CLUSTER_SPOT_LIGHTS" value="21" enum="DebugDraw">
+		<constant name="DEBUG_DRAW_CLUSTER_SPOT_LIGHTS" value="22" enum="DebugDraw">
 		</constant>
-		<constant name="DEBUG_DRAW_CLUSTER_DECALS" value="22" enum="DebugDraw">
+		<constant name="DEBUG_DRAW_CLUSTER_DECALS" value="23" enum="DebugDraw">
 		</constant>
-		<constant name="DEBUG_DRAW_CLUSTER_REFLECTION_PROBES" value="23" enum="DebugDraw">
+		<constant name="DEBUG_DRAW_CLUSTER_REFLECTION_PROBES" value="24" enum="DebugDraw">
 		</constant>
-		<constant name="DEBUG_DRAW_OCCLUDERS" value="24" enum="DebugDraw">
+		<constant name="DEBUG_DRAW_OCCLUDERS" value="25" enum="DebugDraw">
 		</constant>
-		<constant name="DEBUG_DRAW_MOTION_VECTORS" value="25" enum="DebugDraw">
+		<constant name="DEBUG_DRAW_MOTION_VECTORS" value="26" enum="DebugDraw">
 		</constant>
 		<constant name="DEBUG_DRAW_INTERNAL_BUFFER" value="26" enum="DebugDraw">
```

Comment:
This needs to be updated too, you need to run doctool again I think

Reply:
Should be corrected in the latest commit.